test-cases/performance/perf-regression-latency-650gb-with-nemesis.yaml

```diff
@@ -35,3 +35,4 @@ send_email: true
 email_recipients: ["scylla-perf-results@scylladb.com"]
 use_prepared_loaders: true
 use_hdr_cs_histogram: true
+email_subject_postfix: 'latency during operations'
```

Comment:
i can be "calculated" automatically, based on existing content, not need to add such parameter in this test

Reply:
I don't want anymore logic spread across multiple places

Reply:
you don't want logic spread, so let's duplicate parameters in an even more spreaded places

Reply:
yes, to the configuration, as it is in other perf test

Reply:
this specific test (and all of its forms - 3 of them right now) are very different from the other tests, and as such, they use a different report (and a different template), so i don't think it is the right approach...
anyway, this is IMHO, and you are the maintainer, and now the responsible for it.. i don't like your approach, because we are having a unncessary parameter in addition to the existing ones

Reply:
I like the new approach - we can reuse more code. Email subject is something that should be configurable from configuration file (at least in some part like prefix).
Now we can get rid of `LatencyDuringUpgradesPerformanceAnalyzer` which was created only to change email subject.

Reply:
Yeah you have a point there.

I think that which analyzer is used should also be in configuration,

Since we have code selecting them based on job names during the send email stage

Reply:
@soyacz removed the unneeded class